DigitalOffice/Services/CompanyService/Repositories/PositionRepository.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Linq;
+using LT.DigitalOffice.CompanyService.Database;
+using LT.DigitalOffice.CompanyService.Database.Entities;
+using LT.DigitalOffice.CompanyService.Repositories.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace LT.DigitalOffice.CompanyService.Repositories
+{
+    public class PositionRepository : IPositionRepository
+    {
+        private CompanyServiceDbContext dbContext;
+
+        public PositionRepository([FromServices] CompanyServiceDbContext dbContext)
+        {
+            this.dbContext = dbContext;
+        }
+
+        public DbPosition GetUserPosition(Guid id)
+        {
+            var dbPosition = dbContext.Positions.
```

Comment:
Why is that?

Reply:
I search user's position

Reply:
Why not:
`var dbPosition = dbContext.Positions.FirstOrDefault(position => position.UserId == id)`?

Reply:
https://fooobar.com/questions/156611/what-does-include-do-in-linq